client/gutenberg/extensions/subscriptions/save.jsx

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+
+export default function Save( { attributes } ) {
+	const { show_subscribers_total } = attributes;
+	return (
+		<RawHTML
+		>{ `[jetpack_subscription_form title="" subscribe_text="" show_subscribers_total="${ show_subscribers_total }"]` }</RawHTML>
```

Comment:
why do we set the empty `title=""` and the empty `subscribe_text="" ` can we just remove that somehow?